plugins/importexport/users/filter/UserXmlPKPUserFilter.php

```diff
@@ -273,11 +280,15 @@ public function parseUser($node)
             // send USER_REGISTER e-mail only if it is a new inserted/registered user
             // else, if the user already exists, its metadata will not be change (just groups will be re-assigned below)
             if ($password) {
-                $mail = new MailTemplate('USER_REGISTER');
-                $mail->setReplyTo($context->getSetting('contactEmail'), $context->getSetting('contactName'));
-                $mail->assignParams(['recipientUsername' => $user->getUsername(), 'password' => $password, 'recipientName' => $user->getFullName()]);
-                $mail->addRecipient($user->getEmail(), $user->getFullName());
-                $mail->send();
+                $template = Repo::emailTemplate()->getByKey($context->getId(), UserCreated::getEmailTemplateKey());
+                $mailable = new UserCreated($context);
+                $mailable
+                    ->recipients($user)
```

Comment:
I think this needs to be an array, `[$user]`.

Reply:
This mailable accepts only the User object as a recipient.